src/app/auth/account/account.component.ts

```diff
@@ -67,6 +69,16 @@ export class AccountComponent extends BaseComponent implements OnInit {
 
         this.user$ = this.pipelineAccountByName(accountName$);
         this.datasetsAccount$ = this.pipelineAccountDatasets(this.user$, queryParams$);
+        this.accountListFlows$ = this.user$.pipe(
+            switchMap((account) => {
+                return this.accountService.getAccountListFlows({
+                    accounName: account.accountName,
+                    page: 0,
+                    perPage: 0,
+                    filters: { byFlowType: null, byStatus: null, byInitiator: null, byDatasetIds: [] },
+                });
+            }),
+        );
```

Comment:
This would kill the server. as it's loading all flows of all datasets of this user. 

Btw, is this data updating?

Reply:
If we cannot live without a counter in UI (maybe we can), then you need a dedicated API query to get the counter. It's a much simpler query, esp. without pagination and filters, and it should not be loading any flow on the server to answer it.

Reply:
Removed this code